### PR TITLE
Add Google Photos to example

### PR DESCRIPTION
**Pull request by Murderlon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
 		"@uppy/dropbox": "latest",
 		"@uppy/golden-retriever": "^3.0.2",
 		"@uppy/google-drive": "latest",
+		"@uppy/google-photos": "^0.1.0",
```
Comment (aduh95) [inline package.json:67]:
```suggestion
		"@uppy/google-photos": "latest",
```

File: yarn.lock [hunk 2]
```diff
@@ -16092,6 +16092,7 @@ __metadata:
     "@uppy/dropbox": "npm:latest"
     "@uppy/golden-retriever": "npm:^3.0.2"
     "@uppy/google-drive": "npm:latest"
+    "@uppy/google-photos": "npm:^0.1.0"
```
Comment (aduh95) [inline yarn.lock:16095]:
```suggestion
    "@uppy/google-photos": "npm:latest"
```

**Comment by aduh95** [comment]
Getting `this.plugin.uppy.validateAggregateRestrictions is not a function` when trying to use the plugin with `yarn dev`

**Comment by Murderlon** [comment]
Works on my machine

**Review by aduh95 — APPROVED** [review]

**Pull request merged**